worldedit-bukkit/adapters/adapter-1_21_3/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_21_3/nbt/PaperweightLazyCompoundTag.java

```diff
@@ -0,0 +1,161 @@
+package com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_21_3.nbt;
+
+import com.sk89q.jnbt.CompoundTag;
+import com.sk89q.jnbt.LazyCompoundTag;
+import com.sk89q.jnbt.ListTag;
+import com.sk89q.jnbt.StringTag;
+import com.sk89q.jnbt.Tag;
+import com.sk89q.worldedit.bukkit.WorldEditPlugin;
+import net.minecraft.nbt.NumericTag;
+import org.enginehub.linbus.tree.LinCompoundTag;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class PaperweightLazyCompoundTag extends LazyCompoundTag {
```

Comment:
I think we're not actually creating those anymore (just in one method that already requires an instance...) so we can probably get rid of that class?